src/client/sandbox/native-methods.ts

```diff
@@ -1,5 +1,5 @@
 /*global Document, Window */
-import getGlobalContextInfo from '../utils/global-context-info';
+import globalContextInfo from '../utils/global-context-info';
```

Comment:
I guess it's better to change the `getGlobalContextInfo` function in the following way

```js
const globalContenxtInfo = getGlobalContextInfoInternal () // calculate once

export default function getGlobalContextInfo () {
    return globalContenxtInfo;
}
```